### ISSUE TITLE
project was build with gradle, instruction to build is using maven, which doesn't work

### ISSUE THREAD

**Issue by jli226688** [spring-guides/top-spring-on-kubernetes#5]
After "curl https://start.spring.io/starter.tgz -d dependencies=webflux,actuator | tar -xzvf -"

The project only has gradle build settings. No maven at all. So the build image command 
"$ ./mvnw spring-boot:build-image -Dspring-boot.build-image.imageName=spring-k8s/gs-spring-boot-k8s"
would not work.

**Comment by robertmcnees** [comment]
Hi @jli226688. You're correct that the link provided in the `curl` example provides only a Gradle project and not maven. The default build system on start.spring.io was switched to Gradle recently. 
We should make sure that the link here generates a maven project, as the rest of the learning content and code is a maven project.

**Comment by robertmcnees** [comment]
Adding `-d type=maven-project` to the URL generates a Maven project. That would make the whole url:

```
curl https://start.spring.io/starter.tgz -d dependencies=webflux,actuator -d type=maven-project | tar -xzvf -
```

I'll mark this as a good issue for a first time contribution and a PR. Thanks for finding it.

**Comment by ryanjbaxter** [comment]
Fixed via #7
